src/Sfa.Tl.Matching.Application/Mappers/PlacementInformationMapper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using AutoMapper;
+using Sfa.Tl.Matching.Domain.Models;
+using Sfa.Tl.Matching.Models.ViewModel;
+
+namespace Sfa.Tl.Matching.Application.Mappers
+{
+    public class PlacementInformationMapper : Profile
+    {
+        public PlacementInformationMapper()
+        {
+            CreateMap<PlacementInformationViewModel, Opportunity>()
+                .ForMember(m => m.Id, config => config.Ignore())
+                .ForMember(m => m.PostCode, config => config.Ignore())
+                .ForMember(m => m.RouteId, config => config.Ignore())
+                .ForMember(m => m.Distance, config => config.Ignore())
```

Comment:
if these three are ignored then can the be removed from view model?

Reply:
Adam has confirmed these are needed for the back button